homework/src/app.js

```diff
@@ -8,4 +8,145 @@
   /* OPERATORS */
 
   /* QUESTIONS FROM HYF REPOS */
-}
\ No newline at end of file
+}
+
+  
+
+  //STRINGS
+  //Add the string to your file and log it.
+  const myString = "hello,this,is,a,difficult,to,read,sentence";
+  console.log(myString);
+  
+  //Log the length of myString.
+  console.log(myString.length);
+  
+  //The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces.
+  strNew = myString.replace(/,/gi, " ");
+  console.log(strNew);
+  
+  //Consider the following string:
+  const str = 'dlroW olleH';
+  array1 = str.split('');
+  array2 = array1.reverse();
+  string = array2.join('');
+  console.log(string);
+  
+  
+  
+  
+  
+  
+  //Arrays
+  const favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+  //Add a statement that adds your favorite animal say 'turtle' to the existing array.
+  favoriteAnimals.push('turtle');
+  //my new array will be ["blowfish", "capricorn", "giraffe", "turtle"]
+  
+  //Log your new array.
+  console.log(favoriteAnimals);
+  
+  //Now add another favorite animal to the array but make sure it is placed after 'blowfish' and before 'capricorn'.
+  favoriteAnimals.splice(1, 0, 'dog');
+  //my new array will be now ["blowfish", "dog", "capricorn", "giraffe", "turtle"]
+  
+  //Log your new array.
+  console.log(favoriteAnimals);
+  
+  //Delete 'giraffe' from the array.
+  favoriteAnimals.splice(3, 1);
+  //the array will be ["blowfish", "dog", "capricorn", "turtle"]
+  console.log(favoriteAnimals)
+  
+  
+  
+  
+  
+  
+  //QUESTIONS FROM HYF REPOS
+  
+  
+  //8.1 First declare at least four variables and assign them different data types.
+  let firstItem = 0;
+  let secItem = "HackYourFuture";
+  let thirdItem = 99.99;
+  let fourthItem = true;
+  
```

Comment:
Almost. In JavaScript there is no difference between Integers and Floats so 0 is of type number as is 99.99. In other languages this would indeed be different.

Reply:
Actually that's why I used 99.99 to show that there is no difference!